privacy/component.go

```diff
@@ -43,7 +43,6 @@ func ParseComponent(v string) (Component, error) {
 
 	if len(split) == 1 {
 		return Component{
-			Type: ComponentTypeBidder,
```

Comment:
Why should this be removed? The default type is "bidder" when none other is specified.